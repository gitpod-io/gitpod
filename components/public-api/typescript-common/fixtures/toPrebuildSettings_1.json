{
  "enable": true,
  "branchMatchingPattern": "main",
  "branchStrategy": "default-branch",
  "prebuildInterval": 42,
  "workspaceClass": "dev"
}
