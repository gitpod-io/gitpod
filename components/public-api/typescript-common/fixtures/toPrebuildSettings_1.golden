{
    "result": {
        "enabled": true,
        "branchMatchingPattern": "main",
        "branchStrategy": "BRANCH_MATCHING_STRATEGY_DEFAULT_BRANCH",
        "prebuildInterval": 42,
        "workspaceClass": "dev"
    },
    "err": ""
}
